TypeCobol/Compiler/Directives/IBMCompilerOptions.cs

```diff
@@ -692,6 +707,14 @@ public bool TrySetIBMOptionStatusAndValue(string optionWord, string? optionParam
                 return false;
             }
         }
+
+        /// <summary>
+        /// If optionWord is a deprecated option this method returns the corresponding warning message to be displayed
+        /// </summary>
+        public bool IsOptionDeprecated(string optionWord, out string? warningMessage)
```

Comment:
You can remove the `?` on `out` param and use the `MaybeNullWhen` attribute.

Reply:
Done